packages/react-components/react-tree/src/components/BaseTreeItem/BaseTreeItem.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { useBaseTreeItem_unstable } from './useBaseTreeItem';
+import { renderBaseTreeItem_unstable } from './renderBaseTreeItem';
+import { useBaseTreeItemStyles_unstable } from './useBaseTreeItemStyles';
+import type { BaseTreeItemProps } from './BaseTreeItem.types';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
+
+/**
+ * BaseTreeItem component - Represents a single node on the Tree
+ */
+export const BaseTreeItem: ForwardRefComponent<BaseTreeItemProps> = React.forwardRef((props, ref) => {
```

Comment:
Why is `BaseTreeItem` necessary? it's not clear from the change list and there's nothing about it in the PR description

Reply:
Yeah, we'll add a proper PR description. the `BaseTreeItem` would be a `TreeItem` with no opinion about styles nor slots, it only cares about tree navigation and tree functionalities, meanwhile `TreeItem` follows design spec.

Reply:
The `BseTreeItem` is necessary for composition reusability as there'll also be a `PersonaTreeItem` that will reuse `BaseTreeItem` as it's sloting and  styles are different.

Reply:
do we have a requirement to have a styleless/slotless treeitem?

Reply:
We don't. I'm providing nonetheless 😬